modules/rgbd/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ if(NOT HAVE_EIGEN)
   message(STATUS "rgbd: Eigen support is disabled. Eigen is Required for Posegraph optimization")
 endif()
 
-if(HAVE_OPENGL)
-  ocv_target_link_libraries(${the_module} PRIVATE "${OPENGL_LIBRARIES}")
+find_package(GLEW REQUIRED)
```

Comment:
It'll fail module build, if GLEW is not found by some reason. "REQUIRED" -> "QUIET"